### PR TITLE
Add "benchmark" option

### PR DESCRIPTION
**Pull request by zazabe**

### CODE REVIEW

**Comment by zazabe** [comment]
@fauvel please review

File: lib/stream/concat.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ module.exports = function(patterns, withSourceMaps) {
         joinedFile.sourceMap = JSON.parse(result.sourceMap);
       }
       streamOut.push(joinedFile);
+      streamOut.push(null);
```
Comment (fauvel) [inline lib/stream/concat.js]: Does this append a line break?

**Comment by fauvel** [comment]
lgtm!

File: lib/stream/concat.js (hunk 1 above)
Comment (zazabe) [inline lib/stream/concat.js]: I will close the stream, like in this example: https://nodejs.org/api/stream.html#stream_an_example_counting_stream

Comment (zazabe) [inline lib/stream/concat.js]: https://nodejs.org/api/stream.html#stream_readable_push_chunk_encoding ;)

**Pull request merged**